gpflow/misc.py

```diff
@@ -251,5 +251,22 @@ def _get_tensor_safe(name, index, graph):
     except KeyError:
         return None
 
+
+def slice_final_dim(X: tf.Tensor, sl: slice, dim: int):
```

Comment:
what is `dim` for? it is not used in function

Reply:
Turns out this whole method can be removed